### ISSUE TITLE
Can't unparse when no data in the object

### ISSUE THREAD

**Issue by gaetanHautecoeur** [parsecsv/parsecsv-for-php#136]
`
$fieldsInfos = [
    'a' => 'AA',
    'b' => 'BB'
];
`
`
$data = [[
    'a' => 'value1',
    'b' => 'value2',
]];
`
`
$csvParser = new Csv();
`
`
$csvParser->unparse($data, $fieldsInfos);
`

is not working and need to set $csvParser->data with the $data to work. Is it the wrong way to use unparse ? 
By the way there is no example of this method in the documentation. 

Moreover the test running unparse is starting with a .csv, does it make sense ?

**Comment by williamknauss** [comment]
Unparse will return to you a single string. I think that was one of the methods in the original class that was never re-factored or removed. 

```php
<?php

include 'vendor/autoload.php';

$fieldsInfos = array('a' => 'AA', 'b' => 'BB');
$data        = array('a' => 'value1', 'b' => 'value2');
$csvParser   = new ParseCsv\Csv();
$csvstring   = $csvParser->unparse($data, $fieldsInfos);
```

@Fonata  - Should this method be removed or refactored to return a populated Csv object?

**Comment by Fonata** [comment]
The general idea of ``unparse`` is to return a string, and that is fine. The name took me a while to wrap my head around, though, but to me it's an ok naming choice that becomes obvious after a little thinking.

However, from just reading the issue here, it seems the parameters are broken. I will look into it and add a test.

**Comment by Fonata** [comment]
All current unparse tests start with a CSV file just to compare the string.

This issue was fixed by #133.
